admin/app/components/complementary-certifications/attach-badges/target-profile-selector/searchbar.js

```diff
@@ -1,9 +1,10 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import ENV from 'pix-admin/config/environment';
```

Comment:
Je mettrais plutôt config quand on ne va pas directement prendre une var env (il y a les 2 exemples dans le code base )

```suggestion
import config from 'pix-admin/config/environment';
```

Reply:
Je n'avais pas su choisir justement !